tldw_Server_API/tests/Agent_Client_Protocol/test_mcp_streamable_http_transport.py

```
@@ -0,0 +1,217 @@
+
+from tldw_Server_API.app.core.Agent_Client_Protocol.adapters.mcp_transports.streamable_http import (
+    MCPStreamableHTTPTransport,
+)
+
+pytestmark = pytest.mark.unit
+
+
+# ---- Helpers ----
+
+
+def make_json_response(result: dict, id: str = "1") -> MagicMock:
```

Ruff (0.15.6)

[error] Function argument `id` is shadowing a Python builtin

(A002)